pkg/network/usm/monitor.go

```diff
@@ -49,10 +50,10 @@ type Monitor struct {
 	httpConsumer    *events.Consumer
 	http2Consumer   *events.Consumer
 	ebpfProgram     *ebpfProgram
-	httpTelemetry   *telemetry
-	http2Telemetry  *telemetry
-	statkeeper      *httpStatKeeper
-	http2Statkeeper *httpStatKeeper
+	httpTelemetry   *http.Telemetry
+	http2Telemetry  *http.Telemetry
+	statkeeper      *http.HttpStatKeeper
```

Comment:
consider changing it to `httpStatKeeper` to have a similar convention to `http2StatKeeper`